.travis.yml

```diff
@@ -5,19 +5,13 @@ cache:
   - directories:
     - vendor/bundle
 rvm:
-  - 2.2.10
-  - 2.3.8
-  - 2.4.5
```

Comment:
I'd like to keep 2.4.5 since that is supported by Rails 5.2.

Reply:
I put them back and configured the travis configuration file to only test these rubies with Rails 5.2.

Reply:
Oh wow I didn't realize Rails 5.2 actually supports Ruby 2.2.